addons/skin.confluence/720p/MyMusicNav.xml

```diff
@@ -148,6 +148,14 @@
 					<alttexturenofocus>-</alttexturenofocus>
 					<usealttexture>library.isscanningmusic</usealttexture>
 				</control>
+				<control type="button" id="204">
+					<description>Get more</description>
+					<textwidth>235</textwidth>
+					<include>ButtonCommonValues</include>
+					<label>21452</label>
+					<onclick>ActivateWindow(addonbrowser,addons://all/xbmc.addon.audio/,return)</onclick>
```

Comment:
Please use `addons://more/audio` (etc) paths similar to addons://sources/*. The 'all' paths are intended for internal use